CMP1903_A1_2324/Testing.cs

```diff
@@ -16,5 +17,31 @@ internal class Testing
          */
 
         //Method
+        public void Test()
+        {
+            // Runs a test versio of the game, and checks the outputs of the rolls
+            Game testGame = new Game();
+            testGame.ExecuteGame();
+
+            if (testGame.TotalOfRolls() < 18)
+            {
+                if (testGame.TotalOfRolls() > 3)
+                {
+                    Console.WriteLine("Total Is As Expected: " + testGame.TotalOfRolls());
```

Comment:
Could verify that individual dice rolls summed equals the value of the TotalOfRolls in error checking

Reply:
Useful to know, the code is now changed so it can check it against previously summed up numbers.